cypress/plugins/ignore-x-frame-headers/manifest.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This is a plugin this is suggested as a workaround for the issue https://github.com/cypress-io/cypress/issues/4220

We would like to have it removed when cypress fixes this issue. 

The issue basically is that if you navigate to a different origin while doing a cypress test cypress wont allow this. 
An example is for us a navigation from `localhost` to the tracker capture app when a find is successful.

Reply:
I'm the author of the "Ignore X-Frame headers" extension copy-pasted in this commit.

I'm happy that the extension is useful to this project, but FYI the extension is not in the public domain and the [source code](https://github.com/guilryder/chrome-extensions/tree/master/xframe_ignore) has a [license](https://github.com/guilryder/chrome-extensions/blob/master/license.txt).

The extension was GPLv3-licensed when the commit was submitted, which **does not seem directly compatible** with the [BSD license of this project](https://github.com/dhis2/capture-app/blob/master/LICENSE).

Since then I've switched the extension to the MIT license upon request by a reader of https://github.com/cypress-io/cypress/issues/4220.

I think the new MIT license is compatible with BSD, but this **doesn't remove the requirement for proper attribution**.

Reply:
Hey @guilryder thanks for writing this extensions. Its been very helpful :) What would be the steps to follow in order to achieve proper attribution while using your extension?

Reply:
There's no strict rule, but copying the license file along with the code is enough.

Some projects also reference all the licensed libraries they use in their about/credit page.

[Reference 1](https://opensource.stackexchange.com/questions/5828/where-would-i-give-credit-to-an-mit-licensed-project-in-an-ios-app) - [Reference 2](https://opensource.stackexchange.com/questions/6838/how-to-properly-re-license-code-released-under-the-mit-license)